lib/scala/logging-truffle-connector/src/main/scala/org/enso/truffleloggerwrapper/TruffleLoggerWrapper.scala

```diff
@@ -22,8 +22,11 @@ class TruffleLoggerWrapper(name: String, masking: Masking) extends Logger {
 
   override def getName: String = underlying.getName
 
-  private def isEnabled(level: Level): Boolean =
+  private def isEnabled(level: Level): Boolean = try {
     underlying.isLoggable(level)
+  } catch {
```

Comment:
There is a [runtime-instrument-common/testOnly *ChangesetBuilderTest](https://github.com/enso-org/enso/actions/runs/12350627142/job/34463903039?pr=11873#step:7:8610) failure which gets fixed by b4f3973034fc2f49dbc2abd2fe5bed7ad9efd9cb